docs/using-airbyte/core-concepts/readme.md

```diff
@@ -87,22 +87,7 @@ Read more about each [sync mode](/using-airbyte/core-concepts/sync-modes/README.
 
 ## Typing and Deduping
 
-Typing and deduping ensures the data emitted from sources is written into the correct type-cast relational columns and only contains unique records. Typing and deduping is only relevant for the following relational database & warehouse destinations:
-
-- Snowflake
-- BigQuery
-
-:::info
-Typing and Deduping is the default method of transforming datasets within data warehouse and database destinations after they've been replicated. We are retaining documentation about normalization to support legacy destinations.
-:::
-
-For more details, see our [Typing & Deduping documentation](/using-airbyte/core-concepts/typing-deduping).
-
-## Basic Normalization
-
-Basic Normalization transforms data after a sync to denest columns into their own tables. Note that normalization is only available for relational database & warehouse destinations that have not yet migrated to Destinations V2, and will eventually be fully deprecated.
-
-For more details, see our [Basic Normalization documentation](/using-airbyte/core-concepts/basic-normalization.md).
+Typing and deduping ensures the data emitted from sources is written into the correct type-cast relational columns and only contains unique records. Typing and deduping is only relevant for the following relational database & warehouse destinations. For more details, see our [Typing & Deduping documentation](/using-airbyte/core-concepts/typing-deduping).
```

Comment:
"Typing and deduping is only relevant for the following relational database & warehouse destinations." It looks like the list this refers to is removed. Should this read "Typing and deduping is only relevant when working with Snowflake and BigQuery"?